libraries/points/README.md

```diff
@@ -1,26 +1,12 @@
 # Points library
 
-## Installation
-
-TODO
-
-## Usage
+Simple library to interact with points on Ceramic using ComposeDB.
 
-### Single point vs multiple points per document
-
-**Symbols**
-
-🟢 built-in support  
-🟡 supported by configuration or other APIs  
-🟠 requires application logic  
-🔴 implementation constraint
+## Installation
 
```

Comment:
do we not want to explain about the two models, or we'll do that differently?

Reply:
I removed the single point model so the comparison table shouldn't be relevant anymore